backend/src/backend/constants.py

```diff
@@ -9,17 +9,17 @@
 
 COLLECTION_EXPIRE_AFTER = datetime.timedelta(days=7)
 
-if not os.getenv("POSTGRES_URI"):
-    raise EnvironmentError("Please set the POSTGRES_URI environment variable")
-else:
-    POSTGRES_URI = os.getenv("POSTGRES_URI")
 
 
 @dataclass
 class BackendConf:
     """
     Backend configuration, read from environment variables and set default values.
     """
+    if not os.getenv("POSTGRES_URI"):
```

Comment:
Leave it above ; this is a class declaration.
We just want to fail early should the env not be set and we know constants.py will be imported anyway